### PR TITLE
Reorg indices

### PR DESCRIPTION
**Pull request by delvedor**
Related: #409.

### CODE REVIEW

File: specification/cluster/_types/ClusterStateBlocksIndex.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { IndicesRolloverConditions } from '@indices/rollover/IndicesRolloverConditions'
+import { RolloverConditions } from '@indices/rollover/types'
```
Comment (philkra) [inline specification/cluster/_types/ClusterStateBlocksIndex.ts:20]: shouldn't `RolloverConditions` being in `indices/_types` if it's used in more than one endpoint?

File: specification/nodes/_types/Stats.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-import { IndexStats } from '@indices/stats/IndexStats'
+import { IndexStats } from '@indices/stats/types'
```
Comment (philkra) [inline specification/nodes/_types/Stats.ts:20]: same here, especially if used from a different namespace, should go into `indices/_types`

**Review by philkra — CHANGES_REQUESTED** [review]

File: specification/nodes/_types/Stats.ts (hunk 2 above)
Comment (delvedor) [reply specification/nodes/_types/Stats.ts:20]: It's used only here :)

File: specification/cluster/_types/ClusterStateBlocksIndex.ts (hunk 1 above)
Comment (delvedor) [reply specification/cluster/_types/ClusterStateBlocksIndex.ts:20]: It's used only there (and cluster) :)

**Review by philkra — APPROVED** [review]
LGTM

**Pull request merged**